oktaawscli/okta_auth.py

```diff
@@ -171,7 +173,7 @@ def verify_single_factor(self, factor, state_token):
                 challenge['keyHandle'] = resp_json['_embedded']['factor']['profile']['credentialId']
                 challenge['challenge'] = resp_json['_embedded']['factor']['_embedded']['challenge']['nonce']
 
-                print("Please touch your U2F device...")
+                self.logger.info("Please touch your U2F device...")
```

Comment:
I'm on the fence about this one too. This is another normal operational message that the user should see, but they'd need to have the `--verbose` flag set in order to. They would otherwise need to notice the light flashing on their U2F token (assuming it has a light).

Reply:
what about a separate logging channel with formatter for user intended messages that strips the level? that would preserve the existing output format as well.